Scripts/Python/xDialogStartUp.py

```diff
@@ -301,6 +301,17 @@ def OnGUINotify(self,id,control,event):
             elif event == kInterestingEvent: ## RollOver Event ##
                 self.ToggleColor(GUIDiag4b, tagID)
 
+            elif event == kSpecialAction:
+                if tagID in gExp_HotSpot or tagID in gVis_HotSpot:
+                    if gSelectedSlot and gPlayerList[gSelectedSlot-gMinusExplorer]:
+                        PtShowDialog("GUIDialog06a")
```

Comment:
GUIDialog06a is the "Linking..." popup?

Reply:
Yes, this is just copy pasta'd from the code executed by the regular play button.